src/registrar/utility/email.py

```diff
@@ -20,7 +22,7 @@ def send_templated_email(template_name: str, subject_template_name: str, to_addr
     context as Django's HTML templates. context gives additional information
     that the template may use.
     """
-
+    logger.info(f"An email was sent! Template name: {template_name} to {to_address}")
```

Comment:
This line is temporary and will be removed. This is for testing purposes only so we know when an email is sent. If this logs in any other context outside of tests on startup, then we are sending emails when we shouldnt be

Reply:
What do you mean by tests on startup?

Reply:
@rachidatecs Tests **or** startup